### DISCUSSION TITLE
Is there a shorter way to compute a count in Adonis 5?

### DISCUSSION THREAD

**Discussion by LeCoupa** [adonisjs/core#3086] [Help] [answered]
I was wondering if there is a shorter way to compute a count? I end up writing long sentences that are hard to remember for such a simple task, so I  may be wrong. 😅

```typescript
const total = (await list.related('companies').query().pojo<{ 'count(*)': number }>().count('*'))[0]['count(*)']
```

**Comment by arthurcvm** [comment, marked as answer]
Hi @LeCoupa,
if you only use in controller you can make this:
```ts
const users = await User.query().withCount('posts')

users.forEach((user) => {
  console.log(user.$extras.postsCount)
})
```
Look the [docs](https://docs.adonisjs.com/reference/orm/query-builder#withcount)

Or you can transform the computed count in attribute of model, look:

```ts
@computed()
  public get postsCount (): number {
    return this.$extras.postsCount
  }
```
